### DISCUSSION TITLE
Последовательные вызовы

### DISCUSSION THREAD

**Discussion by Naturalov** [aiogram/aiogram#1377] [Q&A] [answered]
### aiogram version

3.x

### Problem

Столкнулся с проблемой что после вызова good_result_menu, нужно сразу отправить "start menu", а подобного функционала нет. 
Единственный вариант явно передавать dji в функцию.
Было бы не плохо сделать вариант последовательного вызова.

### Possible solution

@router.message(id='start_menu', SomeFilter())
async def start_menu(some_dji):
    ....
    
    
@router.message(SomeAnotherFilter())
async def another_menu(some_dji):
    await bot.reccurent_update(id='start_menu')

**Comment by JrooTJunior** [comment]
What do you mean? The given example and wording are not clear, explain in more detail and, of course, in English.

**Comment by Naturalov** [comment]
> What do you mean? The given example and wording are not clear, explain in more detail and, of course, in English.

Sure, i mean that it would be great if we can feed "fake" update. For example we need after "action_finish_menu" to send "start_menu". We can or run again function, or make a "fake" update with the same parameters. For example we had message with message_id=5, and we need to send 2 menus, finish action and send start menu, we are doing finish_action func, and then call start_menu with the same update, but calling another function

**Comment by JrooTJunior** [comment, marked as answer]
You can call function directly.

Based on your example:
```python
@router.message(SomeAnotherFilter())
async def another_menu(some_dji):
    await start_menu(some_dji)
```
